### PR TITLE
Fix JwtAuthExtension initialization & complete Django 6 migration

### PR DESCRIPTION
**Pull request by AswinJ1**
This update addresses critical lifecycle initialization bugs within the Strawberry GraphQL JWT Auth Extension, completes the Django 6 deprecation migration path, and implements modern Python timezone/datetime standards.

##Changes Made

### 1. Strawberry Extension State Lifecycle Fixes (`chowkidar/extension.py`)
- **Restored `__init__` Initialization**: Added back the `__init__` constructor and routed it to populate the base state variables (`user`, `_request`, `_new_jwt_access_token`, etc.) upon instantiation via `self._request_state()`. This prevents the `AttributeError: object has no attribute` that occurs when Strawberry resolves fields before `on_request_start()` completes.
- **Defensive Guards in `resolve()`**: Added `hasattr` checks directly in `resolve()` to ensure that even if the extension lifecycle is completely bypassed (e.g. during specific complex field resolution schemas or test edge-cases), the authentication variables safely default to `None` or `False`. 

### 2. Django 6 Database Constraints (`chowkidar/models.py`)
- **Deprecated `unique_together`**: Removed the legacy `unique_together` Meta attribute.
- **Adopted `UniqueConstraint`**: Replaced it with the modern Django `UniqueConstraint` structure. Because `AbstractRefreshToken` is an abstract model, string interpolation `%(app_label)s_%(class)s_unique_token_revoked` was utilized in the constraint name to ensure child classes inherit unique constraint names in the database.

### 3. Timezone & Datetime Modernization (`chowkidar/settings.py`, `chowkidar/utils/jwt.py`)
- **`django.utils.timezone.timedelta` Removal**: Replaced all usages of the deprecated Django `timedelta` alias with Python's standard `from datetime import timedelta`.
- **`datetime.utcnow()` Deprecation**: Replaced `datetime.utcnow()` with the timezone-aware `django.utils.timezone.now()` to comply with standard Python 3.12+ warnings and Django best practices.
- **JWT Algorithms**: Extracted algorithm strings into the centralized `JWT_ALGORITHM` setting.

### 4. Package Infrastructure (`setup.cfg`)
- Officially expanded package support identifiers. Added classifiers for:
  - Python `3.10` through `3.13`
  - Django `4.0` through `6.0`

### 5. Testing Enhancements (`tests/test_extension.py`)
- Implemented robust new tests including `test_init_sets_state_via_init_after_init` and `test_resolve_defensive_guards` that rigorously verify that context state maintains integrity when edge case execution orders are simulated. 
- The test suite continues to pass at 100% (32/32 tests).

### CODE REVIEW

File: chowkidar/utils/jwt.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import jwt
-from django.utils.timezone import datetime, timedelta
```
Comment (seshagiriprabhu) [inline chowkidar/utils/jwt.py:2]: Use Django timezone as it will read the timezone from the client Django application's settings. 

`from django.utils import timezone`
